### PR TITLE
[WIP] Podcast

### PR DESCRIPTION
**Pull request by SmithPeder**
I've been working on podcasts during the last weeks. Would love some input and suggestions if somebody has any thoughts.

Needs https://github.com/webkom/lego/pull/1300 to work

`List Page`
screenshot 2018-10-24 at 23 06 33

`Edit/add page`
screenshot 2018-10-24 at 23 09 03

### CODE REVIEW

File: app/routes/events/components/EventList.js [hunk 1]
```diff
@@ -124,6 +124,7 @@ type EventListProps = {
   fetchMore: () => Promise<*>
 };
 
+// $FlowFixMe
```
Comment (kotlarz) [inline app/routes/events/components/EventList.js]: Hmmmmm :thinking:
screenshot_20181025-010700

**Comment by SmithPeder** [comment]
This was merged quite fast due to those goddam `yarn.lock` and `flow-typed` configs. I wanted the updated versions in master before I finish this route.

@odinuge How will this affect everybody else? The new redux flow is stricter than the old one, and I had to add that stupid FlowFix just to get the test to pass. I feel like this was not the ideal way to do it.

File: app/routes/events/components/EventList.js (hunk 1 above)
Comment (odinuge) [reply app/routes/events/components/EventList.js]: https://github.com/webkom/lego-webapp/blob/990f636d1694e5df6b6907cf0a9f7f988e940a00/.flowconfig#L24
:tm:

Comment (SmithPeder) [reply app/routes/events/components/EventList.js]: This isn't even my code 👎 
`() => (Not my code )=> (Not my mistake) => (Not my fault) => (Not my job)`

**Comment by odinuge** [comment]
Hey, smooth. That shouldn't affect anyone, so no problems. :smile: 


https://github.com/webkom/lego-webapp/blob/990f636d1694e5df6b6907cf0a9f7f988e940a00/.flowconfig#L24

**Comment by estensen** [comment]
Play button looks slightly off center

**Comment by SmithPeder** [comment]
Yeah, this is one of those visuals mind tricks, where something that is actually in the center looks off center. I tried moving the icon to the right a bit to compensate, but when then when it switches to a pause( || ) icon that one looks off.

**Pull request merged**